sdk/metric/sdk.go

```diff
@@ -247,13 +247,37 @@ func (m *SDK) Labels(kvs ...core.KeyValue) api.LabelSet {
 
 	// Sort and de-duplicate.
 	sort.Stable(&ls.sorted)
-	oi := 1
-	for i := 1; i < len(ls.sorted); i++ {
-		if ls.sorted[i-1].Key == ls.sorted[i].Key {
-			ls.sorted[oi-1] = ls.sorted[i]
+	oi := 0
+	for i := 0; i < len(ls.sorted); i++ {
+		vNew := ls.sorted[i]
+
+		if oi == 0 {
+			if vNew.Value.Empty() {
+				continue
```

Comment:
My reading of this is that if the value is an empty string it will be dropped. Is that the intent?

I seems like (similar to the specified actions to take for tracing attributes) we would want to preserve this. Instead dropping if the value was unspecified or `INVALID`.

Reply:
If an empty string equals unspecified, then I think it's correct to drop.

See this remark: https://github.com/open-telemetry/opentelemetry-specification/issues/345#issuecomment-594382973

Reply:
I intended to pass INVALID values through, but I see that as a different question. These would show up as errors in the exporter.

The specific detail being addressed is that empty-string equals unspecified, which matches Prometheus and I thought was the direction we discussed int he 2/27/20 SIG meeting.

Reply:
Reading further through the related liked PRs and comments I see this is a thoroughly discussed topic. Please disregard.

Reply:
Reading tests were very good for understanding this PR but I would suggest to add some comments to the code itself.